lib/rules/declaration-block-no-duplicate-properties/__tests__/index.js

```diff
@@ -282,7 +282,6 @@ testRule(rule, {
   ruleName,
   config: [true],
   skipBasicChecks: true,
-  syntax: "styled",
```

Comment:
Why `syntax: "jsx"` isn't required here? `postcss-syntax` recognizes it automatically? Won't it produce false positives, i. e. report no warnings even if there are there?

Reply:
`postcss-syntax` will automatically infer syntax based on source code